h/test/queue_test.py

```diff
@@ -77,11 +97,12 @@ def test_get_writer_namespace(fake_nsqd):
 
 @patch('gnsq.Nsqd')
 def test_writer_serializes_dict(fake_nsqd):
-    fake_client = fake_nsqd.return_value
-    settings = {
+    req = DummyRequest()
+    req.registry.settings = {
         'nsq.namespace': 'abc',
     }
-    writer = queue.get_writer(settings)
+    fake_client = fake_nsqd.return_value
+    writer = queue.get_writer(req)
     writer.publish('sometopic', {
         'key': 'value',
     })
```

Comment:
So you've updated the tests to account for taking `request` instead of a settings object, but it would be good to test that the sentry integration actually does what we think, no?